.ci/compute_projects_test.py

```diff
@@ -153,11 +183,13 @@ def test_mlir(self):
         )
         self.assertEqual(env_variables["projects_to_build"], "clang;flang;llvm;mlir")
         self.assertEqual(
-            env_variables["project_check_targets"], "check-flang check-mlir"
+            env_variables["project_check_targets"],
+            "check-flang check-mlir",
         )
         self.assertEqual(env_variables["runtimes_to_build"], "")
         self.assertEqual(env_variables["runtimes_check_targets"], "")
         self.assertEqual(env_variables["runtimes_check_targets_needs_reconfig"], "")
+        self.assertEqual(env_variables["enable_cir"], "ON")
```

Comment:
This needs to be updated after planning to split this out?

Reply:
I don't think so. The thing that I split out was running CIR tests when MLIR is modified. I'd like to keep the build dependency because changes to MLIR core interfaces can and do break the CIR build.

Reply:
Yeah, I can do that. I'll post an update with that and the other changes you asked for shortly.